### ISSUE TITLE
Tests

### ISSUE THREAD

**Issue by PatrickHeneise** [gitevents/core#102]
currently travis is failing as tests have not been updated to 0.3.0.

Would appreciate some help here updating all the tests to the new architecture with talks and events separated.

**Comment by iancrowther** [comment]
@PatrickHeneise - @mike182uk asked for some guidance here

- test what exactly?

@alistairstead maybe this might be a good place for you to jump in?

**Comment by PatrickHeneise** [comment]
> test what exactly?

test all the things

afaik since the last refactor everything is invalid or not working (if they work they're invalid I guess.

**Comment by PatrickHeneise** [comment]
Check out the meetup plugin, tests are working there:

https://github.com/gitevents/gitevents-meetup/blob/master/tests/events.js

**Comment by mike182uk** [comment]
I have made some progress with this in these PR's:

https://github.com/gitevents/core/pull/156
https://github.com/gitevents/core/pull/158
https://github.com/gitevents/core/pull/159

I have removed all of the old tests and created some high level integration tests to cover the modules. The plan is to have all modules covered and maybe even `gitevents.js` covered by high level integration tests. Once we have these tests, it will be easier to refactor / make changes.

Currently the talks and events modules are covered. The only other module in `lib` is `milestones`. In its current state this module is untestable - it returns a promise that is never resolved which means we cannot hook into the code. The code needs to be modified to make it testable.
